packages/components/src/date-time/style.scss

```diff
@@ -111,6 +111,31 @@
 	}
 }
 
+.components-datetime__date .CalendarDay .components-datetime__date__day {
+	height: 100%;
+	display: flex;
+	justify-content: center;
+	align-content: center;
+	flex-direction: column;
+	position: relative;
+
+	&.has-events::before {
+		content: ' ';
```

Comment:
Linter check is not passing due to this:

[3] packages/components/src/date-time/style.scss
[3]  123:12  ✖  Expected double quotes   string-quotes